docs/synapse/userguides/storm_ref_type_specific.ipynb

```diff
@@ -918,9 +923,10 @@
     "**Usage Notes**\n",
     "\n",
     "- Because the asterisk is a non-alphanumeric character, the string to be matched must be enclosed in single or double quotes (see :ref:`whitespace` and :ref:`literals`).\n",
-    "- Because domains are reverse-indexed instead of prefix indexed, partial string matching can only occur based on the end (suffix) of a domain. It is not possible to lift by a string at the beginning of a domain. For example, ``inet:fqdn=\"yahoo*\"`` and ``inet:fqdn^=yahoo`` are both invalid.\n",
-    "- Domains can be lifted or filtered using the regular expression (regex) extended operator (``~=``), though lifting in particular may impose a performance overhead (see :ref:`lift-regex` and :ref:`filter-regex`).\n",
-    "- Suffix matching is not supported for filter operations (note that a \"lift by suffix\" is effectively a combined \"lift and filter\" operation)."
+    "- Because domains are reverse-indexed instead of prefix indexed, for **lift** operations, partial string matching can only occur based on the end (suffix) of a domain. It is not possible to lift by a string at the beginning of a domain. For example, ``inet:fqdn=\"yahoo*\"`` and ``inet:fqdn^=yahoo`` are both invalid.\n",
+    "- Domains can be **filtered** by prefix (``^=``). For example, ``inet:fqdn=\"*.biz\" +inet:fqdn^=smtp`` is valid.\n",
+    "- Doomains cannot be filtered based on suffix matching (note that a \"lift by suffix\" is effectively a combined \"lift and filter\" operation).\n",
```

Comment:
Extra 'o' in Doomains

Reply:
Those are the Domains of Mt. Doom

Reply:
"doomains" = malicious domains.

j/k fixed